packages/wonder-blocks-core/src/hooks/use-unique-id.md

```diff
@@ -0,0 +1,41 @@
+### useUniqueIdWithoutMock (hook)
```

Comment:
Hmm, are you missing docs for `useUniqueIdWithMock`?

Reply:
I need to remove this file.  All the docs are in use-unique-id.stories.mdx.  I couldn't get the hooks examples to work in styleguidist.  I guess I could just have the text parts in the .md file and leave out the examples.

Reply:
I ended up duplicating the docs without the examples from use-unique-id.stories.mdx.